app/app_test.go

```diff
@@ -677,6 +677,20 @@ func (s *S) TestCreateAppProvisionerFailures(c *check.C) {
 	c.Assert(err, check.NotNil)
 }
 
+func (s *S) TestCreateAppUserFromTsuruToken(c *check.C) {
+	user := auth.User{
+		Email: "my-token@token.tsuru.internal",
```

Comment:
Shouldn't this email end with `token.tsuru.invalid` in order to get true from `IsEmailFromTeamToken()`?

Reply:
Sure, I overlooked it.